### DISCUSSION TITLE
1.4.1 BugFixes: Ortho camera view and numpad keys

### DISCUSSION THREAD

**Discussion by JulienHeijmans** [JulienHeijmans/quicksnap#40] [Releases]
Some smaller bugfixes:
* The tool now work properly when using orthographic view from a camera in the scene.
* The numpad keys 1,2,3 can now also be used to switch the snap target type.

<hr /><em>This discussion was created from the release <a href='https://github.com/JulienHeijmans/quicksnap/releases/tag/1_4_1'>1.4.1 BugFixes: Ortho camera view and numpad keys</a>.</em>

**Comment by TinanoumaX** [comment]
Hello, just for say your add on is the simply and the best i have found. If you can make the possibility to rotate on XYZ on the select point will be great.😎
